DigitalLearningSolutions.Data/DataServices/RegistrationDataService.cs

```diff
@@ -174,71 +175,69 @@ OUTPUT Inserted.ID
             return candidateNumber;
         }
 
-        public int RegisterAdmin(AdminRegistrationModel registrationModel)
+        public int RegisterAdmin(AdminRegistrationModel registrationModel, int userId)
         {
-            var values = new
+            using var transaction = new TransactionScope();
+
+            var adminValues = new
             {
-                forename = registrationModel.FirstName,
-                surname = registrationModel.LastName,
-                email = registrationModel.PrimaryEmail,
-                password = registrationModel.PasswordHash,
+                userId,
                 centreID = registrationModel.Centre,
-                categoryId = registrationModel.CategoryId,
-                centreAdmin = registrationModel.IsCentreAdmin,
+                categoryID = registrationModel.CategoryId,
+                isCentreAdmin = registrationModel.IsCentreAdmin,
                 isCentreManager = registrationModel.IsCentreManager,
-                approved = registrationModel.Approved,
                 active = registrationModel.Active,
-                contentCreator = registrationModel.IsContentCreator,
-                contentManager = registrationModel.IsContentManager,
+                isContentCreator = registrationModel.IsContentCreator,
+                isContentManager = registrationModel.IsContentManager,
                 importOnly = registrationModel.ImportOnly,
-                trainer = registrationModel.IsTrainer,
-                supervisor = registrationModel.IsSupervisor,
-                nominatedSupervisor = registrationModel.IsNominatedSupervisor
+                isTrainer = registrationModel.IsTrainer,
+                isSupervisor = registrationModel.IsSupervisor,
+                isNominatedSupervisor = registrationModel.IsNominatedSupervisor,
+                // deprecated columns

```

Comment:
*"Currently non-null deprecated columns" would be clearer

In a few weeks they'll be nullable just like all the other deprecated columns and people will wonder why we're arbitrarily choosing some deprecated columns to update